weave-js/src/components/PagePanelComponents/Home/HomeCenterEntityBrowser.tsx

```diff
@@ -1,12 +1,6 @@
 import React, {useCallback, useEffect, useMemo, useState} from 'react';
 
-import {
-  IconAddNew,
-  IconCopy,
-  IconDown,
-  IconInfo,
-  IconOpenNewTab,
-} from '../../Panel2/Icons';
+import {IconCopy, IconDown, IconInfo, IconOpenNewTab} from '../../Panel2/Icons';
```

Comment:
We should try updating to use `@wandb/weave/components/Icon` as we go, but ok if you want to leave that for later.

Reply:
updated in next commit